docs/content/en/docs/tasks/cluster/cluster-upgrades/baremetal-upgrades.md

```diff
@@ -16,6 +16,21 @@ The upgrade command also upgrades core components of EKS Anywhere and lets the u
 >**_NOTE:_** Currently only Minor Version Upgrades are support for Bare Metal clusters. No other aspects of the cluster upgrades are currently supported.
 >
 
+{{% alert title="Important" color="warning" %}}
+
+In `eksctl anywhere` version `v0.15.0`, we introduced the full lifecycle controller to fully manage new workload clusters.
+The controller behavior would roll out new nodes in the workload cluster whenever the user upgraded the management cluster and its management components.
+
+In `eksctl anywhere` version `v0.16.0`, we changed this behavior to allow users to be explicit when deciding which clusters to upgrade.
+Therefore, workload clusters are no longer affected by management cluster upgrades.
+Due to this change, each cluster must be individually upgraded to enjoy the latest features on all clusters.
+If you have a management cluster running EKS Anywhere version v0.15, you can successfully upgrade to EKS Anywhere version v0.16 and observe no changes to any of its workload clusters.
+
+When triggering a workload cluster upgrade after upgrading the management cluster, please keep in mind that it will not only reflect your changes in the cluster spec, but also any new improvements included in the new EKS-A controller.
```

Comment:
```suggestion
When triggering a workload cluster upgrade after upgrading the management cluster, please keep in mind that it will not only reflect your changes in the cluster spec, but also any new improvements included in the new EKS Anywhere controller.
```